client/src/components/PreviousTripsCard/PreviousTripsCard.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import PreviousTripsCard from "./PreviousTripsCard";
+
+storiesOf("PreviousTripsCard", module)
```

Comment:
Instead of including red placeholder divs, try coming up with how this would look, even if the mock isn't provided. It can be as simple as an image card with the name of the trip.